pkg/controller/scyllacluster/sync_statefulsets.go

```diff
@@ -355,7 +355,7 @@ func (scc *Controller) pruneStatefulSets(
 }
 
 // createMissingStatefulSets creates missing StatefulSets.
-// It return true if a statefulSet was created and an error.
+// It return true if done and an error.
```

Comment:
"It return true if done and an error." is really vague and doesn't really help understand the purpose of this function any better than by just looking at its signature.

Reply:
yeah, I guess I could have elaborated on that more then expecting the readers to be familiar with the pattern.